package.json

```diff
@@ -143,9 +144,6 @@
     "> 0.25%",
     "not dead"
   ],
-  "engines": {
-    "node": "22"
-  },
```

Comment:
Single source of truth should be the `.nvmrc` 🙂

Reply:
Right, and this is more relevant for packages, not for an app 🤝